plugins/org.eclipse.fordiac.ide.model.search/src/org/eclipse/fordiac/ide/model/search/ModelSearchPattern.java

```diff
@@ -13,21 +13,17 @@
  *******************************************************************************/
 package org.eclipse.fordiac.ide.model.search;
 
-import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class ModelSearchPattern { // extends SearchPattern
 
-	private final String toTest;
-	private Pattern searchPattern;
-	private final String searchString;
+	private final Pattern searchPattern;
 
-	public ModelSearchPattern(final String toTest, final ModelQuerySpec modelQuerySpec) {
-		this.toTest = toTest;
-		this.searchString = modelQuerySpec.getSearchString();
+	public ModelSearchPattern(final ModelQuerySpec modelQuerySpec) {
+		searchPattern = Pattern.compile(convertSearchStringToPattern(modelQuerySpec.getSearchString()));
```

Comment:
Should the Pattern be case insensitive?

Reply:
Yes it should

Reply:
In my latest PR I didn't touch now the existing search as I was not perfectly sure about the impact on search behavior. I guess this should be investigated next.